### ISSUE TITLE
does not replace nvim command

### ISSUE THREAD

**Issue by Alex23rodriguez** [richin13/asdf-neovim#18]
After installing a certain version and running `asdf global neovim <version>`, the system's `nvim` command doesn't get updated (neither do the `vim` or the `vi` commands).

The only way I can get the installed version to run is with `asdf exec nvim`

**Comment by richin13** [comment]
Hello @Alex23rodriguez ! 

That sounds like your `PATH` might be sorted in a way that prioritizes system binaries rather than `asdf`'s. In order to help you troubleshoot, can you share the output of:

```bash
echo "$PATH"
```

**Comment by Alex23rodriguez** [comment]
Hi! there was something weird going on on my mac, where `asdf` got added to `PATH` _after_ the paths from `/etc/paths` and `/etc/paths.d`. It would consistently happen even if I changed my `.zshrc`. After resetting killing the `tmux` session and restarting my computer the problem went away and things got correctly added to the beginning of the `PATH`.

TLDR: the problem was indeed the `PATH`, and some long running tmux session.
